config/config.exs

```diff
@@ -24,7 +25,10 @@ config :hex_web,
   ses_user:      System.get_env("HEX_SES_USERNAME"),
   ses_pass:      System.get_env("HEX_SES_PASSWORD"),
 
-  secret:        System.get_env("HEX_SECRET")
+  secret:        System.get_env("HEX_SECRET"),
+
+  signing:       signing,
```

Comment:
Lets keep the config as lean as possible and just use `signing_key`. We can check for `nil` on that value.